Win32Emu/Win32/NativeTypes.cs

```diff
@@ -770,7 +770,7 @@ public enum DDPCaps : uint
 		DDPCAPS_4BIT = 0x00000004,
 		DDPCAPS_8BIT = 0x00000008,
 		DDPCAPS_PRIMARYSURFACE = 0x00000002,
-		DDPCAPS_ALLOW256 = 0x00000100,
+		DDPCAPS_ALLOW256 = 0x00000040,
```

Comment:
Can you add the rest of the values in this enum? https://doxygen.reactos.org/d7/de9/sdk_2include_2psdk_2ddraw_8h_source.html

That way we can log the names rather than the values